### ISSUE TITLE
Support for linux aarch64

### ISSUE THREAD

**Issue by aspect** [mattsse/chromiumoxide#238]
How would one go about supporting Linux (Ubuntu in my case) running on top of `aarch64`? 

I naturally endup with `FetcherError(UnsupportedOs("linux", "aarch64"))`

I can try making a chromium build (if someone gives me some guidance) and contribute the build, as well as modify the fetcher...  Can also provide access to such a VM (These VPSes are available from Hetzner and NetCup). 

Would appreciate some input on what the steps would be to accomplish this.

**Comment by Sytten** [comment]
I coded that part. Sadly chromium doesn't have official builds for linux aarch64 so I didnt offer it.
I am looking to support chrome for testing (https://googlechromelabs.github.io/chrome-for-testing/) which has linux aarch64 builds, so that will fix it

**Comment by Sytten** [comment]
I added chrome for testing but they still dont support linux aarch64, this will probably become required at some point since a lot of the cloud infra is moving to arm but for now this is blocked.

**Comment by richarddd** [comment]
Chrome has been available on aarch64 for a while now:
https://packages.debian.org/sid/arm64/chromium/download

**Comment by Sytten** [comment]
Yes but not easily via an API and not via google. OP wanted to have it in the fetcher.
